tests/Unit/NumericalAlgorithms/LinearOperators/Test_Divergence.cpp

```diff
@@ -0,0 +1,238 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Index.hpp"
+#include "DataStructures/IndexIterator.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Domain/CoordinateMaps/AffineMap.hpp"
+#include "Domain/CoordinateMaps/CoordinateMap.hpp"
+#include "Domain/CoordinateMaps/ProductMaps.hpp"
+#include "Domain/LogicalCoordinates.hpp"
+#include "NumericalAlgorithms/LinearOperators/Divergence.hpp"
+#include "NumericalAlgorithms/Spectral/LegendreGaussLobatto.hpp"
+#include "Utilities/MakeArray.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+namespace {
+
+template <size_t Dim>
+struct Flux1 : db::DataBoxTag {
+  using type = tnsr::I<DataVector, Dim, Frame::Grid>;
+  static constexpr db::DataBoxString label = "Vector_t";
```

Comment:
I'm confused by this label

Reply:
I am too

Reply:
I found out while writing tests for the partial derivative compute item that these were just copied from the partial derivative test. I've changed that in #415 for the partial derivative test, not sure why that was there originally, but whatever :)